### PR TITLE
PortX 2.2.6

### PR DESCRIPTION
**Pull request by netsarang-bigseungho**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Review by SMillerDev — APPROVED** [review]

**Review by samford — COMMENTED** [review]
The `livecheck` block makes sense to me. The `latest` dmg file is linked from the [download page](https://portx.online/en/download/) and the hash in the URL (`0ac7ea20`) is the same in the available archive.org snapshots from 2023 (albeit, there are only two to compare). Hopefully the hash doesn't change with website updates but we can address that if/when it happens.

I'll leave overall review to our cask maintainers but it looks fine to me.

File: Casks/p/portx.rb [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-  version "2.2.5,10.52"
-  sha256 arm:   "FEC219835FC4B6A616BE480F3BBBC1569EB396887C69327C5AF9A55F710D0F66",
-         intel: "7FB68D6BCF2381071D49659967E677AA262868702B79C9B058488D36C51B6BC0"
+  version "2.2.6,10.55"
+  sha256 arm:   "181CE3FF7C2E9F8B47A099CC94274204ED0713944F807AA3F532A48F2E6DE560",
+         intel: "510D32F5A57E7CA765D0788F7259C3B5B7D2FC19F509F80451B93520A6FDB149"
 
   url "https://cdn.netsarang.net/0ac7ea20/PortX-#{version.csv.first}-#{arch}.dmg",
-      verified: "cdn.netsarang.net/"
+      verified: "cdn.netsarang.net/0ac7ea20/"
```
Comment (bevanjkay) [inline Casks/p/portx.rb:9]: I don't think we need to add the checksum to the `verified` parameter.
The entire `cdn` is owned by the developer, so can be verified.

**Review by miccal — APPROVED** [review]

**Review by bevanjkay — APPROVED** [review]

**Comment by krehel** [comment]
This is fine for me, but I'd like to avoid `:extract_plist` where possible.

@NetSarangComputer it would be very helpful to have a method to get version information without needing to download 2 * 100mb files.  Please consider publishing that information.

Thanks

**Review by krehel — APPROVED** [review]

**Pull request merged**